awscli/examples/ssm/register-patch-baseline-for-patch-group.rst

```diff
@@ -9,6 +9,6 @@ Command::
 Output::
 
   {
-    "PatchGroup": "Production",
-    "BaselineId": "pb-045f10b4f382baeda"
+    "BaselineId": "pb-045f10b4f382baeda",
+	"PatchGroup": "Production"
```

Comment:
Let's fix the indentation here.

Reply:
Indentation has been fixed in the latest commit